server/test_devpi_server/test_stage_customizer.py

```diff
@@ -157,3 +157,25 @@ def validate_config(self, oldconfig, newconfig):
     assert r.json['message'] == "You can't have fewer bases than before."
     result = mapp.getjson(api.index)
     assert result['result']['bases'] == ['user/foo']
+
+
```

Comment:
a test is missing for succes case of ``on_modified``

Reply:
fixed